dags/plugins/jid.py

```diff
@@ -78,6 +81,23 @@ def __params_to_args__(params):
 
     def __slurm_parameters__(params, task_id, location):
       return __params_to_args__(params) + " --task " + task_id + " --facility " + location
+    
+    def __new_config_path__(config_path, exp_name, branch_id):
+      config_dir, config_file_name = os.path.split(config_path)
+      config_name, file_extension = os.path.splitext(config_file_name)
+
+      new_config_name = f"{exp_name}_sample_{self.branch_id}" + file_extension
+      new_config_dir = os.path.join(config_dir, exp_name + "_init_samples_configs")
```

Comment:
I'm getting a little lost with the path manipulations - should this be in `${root_dir}/${EXP}_init_samples...` or `${root_dir}/bayesian_opt/${task.tag}/${EXP}+_init_samples...` .

The latter is taken from the `bo_init_samples_config` task.

Reply:
Corrected to a common path: ${root_dir}/bayesian_opt/${EXP}+_init_samples (removed ${task.tag})